{"abstract":[{"text":"An unknown error occured while writing an image to a file.","type":"text"}],"variants":[{"paths":["\/documentation\/scanbotdocumentscannersdk\/sbsdkimageprocessorerror\/writeerror"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"role":"symbol","externalID":"s:25ScanbotDocumentScannerSDK24SBSDKImageProcessorErrorO05writeG0yA2CmF","symbolKind":"case","roleHeading":"Case","modules":[{"name":"ScanbotDocumentScannerSDK"}],"title":"SBSDKImageProcessorError.writeError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"writeError"}]},"sections":[],"hierarchy":{"paths":[["doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK","doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK\/SBSDKImageProcessorError"]]},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"writeError","kind":"identifier"}]}],"kind":"declarations"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK\/SBSDKImageProcessorError\/writeError"},"references":{"doc://net.doo.ScanbotDocumentScannerSDK/documentation/ScanbotDocumentScannerSDK/SBSDKImageProcessorError":{"role":"symbol","title":"SBSDKImageProcessorError","type":"topic","url":"\/documentation\/scanbotdocumentscannersdk\/sbsdkimageprocessorerror","identifier":"doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK\/SBSDKImageProcessorError","abstract":[{"text":"An enum of possible errors when processing images.","type":"text"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"SBSDKImageProcessorError","kind":"identifier"}],"kind":"symbol","navigatorTitle":[{"text":"SBSDKImageProcessorError","kind":"identifier"}]},"doc://net.doo.ScanbotDocumentScannerSDK/documentation/ScanbotDocumentScannerSDK/SBSDKImageProcessorError/writeError":{"type":"topic","abstract":[{"type":"text","text":"An unknown error occured while writing an image to a file."}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"writeError","kind":"identifier"}],"url":"\/documentation\/scanbotdocumentscannersdk\/sbsdkimageprocessorerror\/writeerror","kind":"symbol","title":"SBSDKImageProcessorError.writeError","role":"symbol","identifier":"doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK\/SBSDKImageProcessorError\/writeError"},"doc://net.doo.ScanbotDocumentScannerSDK/documentation/ScanbotDocumentScannerSDK":{"abstract":[{"type":"text","text":"Version: 6.2.0"}],"url":"\/documentation\/scanbotdocumentscannersdk","role":"collection","title":"ScanbotDocumentScannerSDK","identifier":"doc:\/\/net.doo.ScanbotDocumentScannerSDK\/documentation\/ScanbotDocumentScannerSDK","kind":"symbol","type":"topic"}}}